tests/modules/test_fillobjects.py

```diff
@@ -0,0 +1,65 @@
+import numpy
+import numpy.testing
+import skimage.morphology
+
+import cellprofiler.image
+import cellprofiler.measurement
+import cellprofiler.modules.fillobjects
+import cellprofiler.object
+
+
+instance = cellprofiler.modules.fillobjects.FillObjects()
+
+labels = numpy.zeros((20, 20), dtype=numpy.uint8)
+
+labels[2:8, 2:8] = 1
+labels[5, 5] = 0
+
+labels[0:8, 12:18] = 2
+labels[2, 15] = 0
+
+labels[12:18, 0:8] = 3
+labels[15, 2] = 0
+
+labels[12:20, 12:20] = 4
+labels[15, 15] = 0
+
+labels = numpy.zeros((9, 20, 20), dtype=numpy.uint8)
+
+labels[0:9, 2:8, 2:8] = 1
+
+labels[0:5, 0:8, 12:18] = 2
+
+labels[4:9, 12:18, 0:8] = 3
+
+labels[1:8, 12:20, 12:20] = 4
+
+
+def test_run(image, module, workspace):
+    module.x_name.value = "example"
```

Comment:
Regarding https://github.com/CellProfiler/CellProfiler/pull/3382#issuecomment-349200052:

[`module.x_name` is an `ObjectNameSubscriber`](https://github.com/CellProfiler/CellProfiler/blob/master/cellprofiler/module.py#L1229-L1232), so you will need to provide the [name of the objects (`"InputObjects"`)](https://github.com/AetherUnbound/CellProfiler/blob/bce15e3e69943921898e966b0574f7139f7f0656/tests/conftest.py#L75) rather than the [name of the image (`"example"`)](https://github.com/AetherUnbound/CellProfiler/blob/bce15e3e69943921898e966b0574f7139f7f0656/tests/conftest.py#L50).

Reply:
You'll also need to set the object's label data (e.g., `objects.segmented = labels`).